### ISSUE TITLE
file:// instead of http://

### ISSUE THREAD

**Issue by hovancik** [HackPlan/quark-shell-mac#39]
a lot of JS scripts use something like: `script.src = '//cdn.syndication.twimg.com/widgets/timelines/'`

which becomes `'file://cdn.syndication.twimg.com/widgets/timelines/'`

would there be a way to fix this, except hardcoding it to http or https?

**Comment by xhacker** [comment]
Good point, but there would not be an easy fix. We will have to bundle a micro web server and serve the static files.
